jest-common/pom.xml

```diff
@@ -69,6 +69,11 @@
         </dependency>
 
         <dependency>
+            <groupId>org.easytesting</groupId>
```

Comment:
I have nothing against fest I'm sure its as awesome as jest if not more :smile:.

Main reason is consistency; having two test classes with a different assert syntax than the rest of the project is just awkward.